crates/sdk/src/connector/example.rs

```diff
@@ -50,13 +50,17 @@ impl Connector for Example {
 
     async fn get_capabilities() -> JsonResponse<models::CapabilitiesResponse> {
         models::CapabilitiesResponse {
-            version: "0.1.1".into(),
+            version: "0.1.3".into(),
```

Comment:
Very unsure about this, I _think_ version is supposed to be the ndc-spec version, is that correct?